spark-connector/v3.3/spark/build.gradle.kts

```diff
@@ -32,22 +32,31 @@ dependencies {
 
   testImplementation(project(":api")) {
     exclude("org.apache.logging.log4j")
+    exclude("org.slf4j")
   }
   testImplementation(project(":clients:client-java")) {
     exclude("org.apache.logging.log4j")
+    exclude("org.slf4j")
   }
   testImplementation(project(":core")) {
     exclude("org.apache.logging.log4j")
+    exclude("org.slf4j")
   }
   testImplementation(project(":common")) {
     exclude("org.apache.logging.log4j")
+    exclude("org.slf4j")
+  }
+  testImplementation(project(":integration-test-common", "testArtifacts")) {
+    exclude("org.apache.logging.log4j")
+    exclude("org.slf4j")
   }
-  testImplementation(project(":integration-test-common", "testArtifacts"))
   testImplementation(project(":server")) {
     exclude("org.apache.logging.log4j")
+    exclude("org.slf4j")
   }
   testImplementation(project(":server-common")) {
     exclude("org.apache.logging.log4j")
+    exclude("org.slf4j")
```

Comment:
why does it work after excluding `slf4j`?

Reply:
The version of `slf4j` is `2.0.9` and that is 1.7.x for Spark connector, after we exclude `slf4j`, it would use 1.7.x

Reply:
got it